### PR TITLE
After last discussion

### PR DESCRIPTION
**Pull request by jezogwza**
Fixed README to include some of the discussion from Thursday.
Added a text file to capture the DAG notions

### CODE REVIEW

File: examples/manifests/README.md [hunk 1]
```diff
@@ -22,39 +47,71 @@ A region manifest will need to define :
...
+- **hw_definition.yaml**
+- **agent_config.yaml**
 
 ----
-## hw_definition.yaml
+## Service Installation
 
-----
-## host_profile.yaml
+There is a set of Core areas of configuration whit respect to this areas .
```
Comment (fmontei) [inline examples/manifests/README.md]: with respect to these areas.

File: examples/manifests/README.md [hunk 2]
```diff
@@ -22,39 +47,71 @@ A region manifest will need to define :
...
+- The under cloud settings themselves. Values that are configurable and might drive the actual deployment of the under cloud.
+- The services configuration and deployment instruction. Services are capabilities provided on top of the under cloud. i.e if we are deploying an AIC Region that will support VM workloads. We will deploy the cLCP, which is an openstack-helm based deployment of openstack.
+
+
+- **core_functions.yaml**
+
+This delivers core component functions that always exist as part of a Container Platform. Componnets that provide functions sich as Logging, and Monitoring. This functions are considered core, and are expected to exist regardless of other domain specific services.
```
Comment (fmontei) [inline examples/manifests/README.md]: such as Logging

File: examples/manifests/README.md [hunk 3]
```diff
@@ -22,39 +47,71 @@ A region manifest will need to define :
...
+
+This is an Armada yaml, that describes the charts of the components that define the cLCP. Openstack components based on openstack-helm, plus other functions that might be found in openstack-helm-infra. 
+
+- **onap.yaml**
+
+This is an Armada yaml, that describes the charts of the components that define an ONAP VMI deployment.
+
+- **<service name>.yaml**
+
+In general we are advocating a naming convention that defines the service name in the yaml file.  The reason for this convention will become apaent when we discuss teh Post Service deployment configuration yaml.
```
Comment (fmontei) [inline examples/manifests/README.md]: will become apparent when we discuss the Post Service

File: examples/manifests/README.md [hunk 4]
```diff
@@ -22,39 +47,71 @@ A region manifest will need to define :
...
 
 ----
-## services.yaml
+## Service Configuration (Post Configuration)
+
+- **<service name>_post.yaml**
+
+This is some sort of YAML , hat provides a generic format that allows teh ability to drive behaviour on the service deployed on top of the UCP/CP platform.
```
Comment (fmontei) [inline examples/manifests/README.md]: This is some sort of YAML that provides a generic format that allows the

File: examples/manifests/README.md [hunk 5]
```diff
@@ -22,39 +47,71 @@ A region manifest will need to define :
...
+This behavior might be API driven, CLI driven or any other mechanism. This TAML should provide generic artifacts that allows the Workflow engine (ShipYard) to identify and execute instructions
```
Comment (fmontei) [inline examples/manifests/README.md]: This YAML

**Review by fmontei — COMMENTED** [review]
Typos

**Comment by bryan-strassner** [comment]
This pull request is fully out of date. Closing.

**Pull request closed without merging**